cloud_run/shelf_server/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use latest stable channel SDK.
```

Comment:
I'm not sure exactly which sample this is, but it's expected we put region tags for the Dockerfile for Run samples.